spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/batch/BatchProperties.java

```diff
@@ -83,6 +89,12 @@ public static class Jdbc {
 		 */
 		private String platform;
 
+		/**
+		 * The length of long string columns in database. Do not override this if you
+		 * haven't modified the schema.
+		 */
+		private Integer maxVarCharLength;
+
```

Comment:
I don't think we should offer this as a property for reasons similar tohttps://github.com/spring-projects/spring-boot/issues/44802#issuecomment-2739655041. We can reconsider should Batch ever provide first-class support for modifying its schema scripts.